main.py

```diff
@@ -11,12 +16,12 @@ def main():
     try:
         connection_parameters = parse_connection_arguments()
     except RuntimeError as ex:
-        print(f'[Err] exception has been caught during argument parsing: {ex}')
+        logger.critical(f'[Err] exception has been caught during argument parsing: {ex}')
 
     try:
         db_connection = DBConnection(connection_parameters)
     except Exception as ex:
-        print(f'[Err] exception has been caught during creating db connection: {ex}')
+        logger.critical(f'[Err] exception has been caught during creating db connection: {ex}')
```

Comment:
Same comments as for necessity of [Err] and exc_info=True parameter.

Reply:
resolved